### DISCUSSION TITLE
Multiple Domain Config in Docker

### DISCUSSION THREAD

**Discussion by yodaphone** [shlinkio/shlink#1073] [Help wanted] [answered]
Can you please let me know how to configure multiple domains in Docker? How do i add more domains to the SHORT_DOMAIN_HOST variable in docker?

**Comment by acelaya** [comment, marked as answer]
You don't need to add more. That env var is just for the default domain.

Just make sure all the domains you want to use resolve your Shlink instance, and you should be good to go.

**Comment by m4ush3r** [comment]
I hope you are doing well @acelaya , about this matter I would like to know how did you configure the domain for the API

[https://api-spec.shlink.io/#/Domains/listDomains](url)

![image](https://user-images.githubusercontent.com/55663719/176335919-73923fb7-6bb2-41f2-b6d4-f3e374ddecc8.png)

Where you list all the domains configured.

It's a bit confusing how this work because neither in my local configuration or aws DB I'm not able to visualize any Domain but it does work with bot domains just like you mentioned **(Just make sure all the domains you want to use resolve your Shlink instance, and you should be good to go**)

![image](https://user-images.githubusercontent.com/55663719/176335110-a0572e19-2002-40e7-b590-a70538e0d94e.png)

**Reply by acelaya** [reply]
https://shlink.io/documentation/advanced/multiple-domains/#external-configuration

That explains what needs to be done outside of Shlink to configure other domains.

As per the empty table, the way to "create" non-default domains on Shlink is by just passing them during short URL creation.

**Reply by ibrahimdevs** [reply]
@acelaya thanks for amazing project! I want to use it on Coolify (Which run on docker), but the documentation only explain normal using. I think it is better to put how to do this external-configuration on Docker. I also opened an discussion on Coolify: https://github.com/coollabsio/coolify/discussions/4474

I believe lots of people using Shlink on Coolify, if you have time, a solution would be amazing.
